src/main/java/Entities/Player.java

```diff
@@ -111,14 +170,21 @@ public String steal(Player victim) {
         }
     }
 
-
+    /**
+     * Method to move the player by x number of steps
+     * @param step the steps to move the player to
+     */
     public void move(int step) {
         position += step;
         if (position > 39) {
             position -= 40;
         }
     }
 
+    /**
```

Comment:
I think this is an outdated version of rollDice, I can substitute this one with a new version and its JavaDocs.

Reply:
Yeah, we've made a couple of changes to the Player class so many of the code here will be changed :) Commit the new changes when you're done